<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondMainWindow</class>
 <widget class="QMainWindow" name="SecondMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>695</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>695</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>OpenHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bank.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>67</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D
</string>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D</string>
    </property>
    <property name="text">
     <string>Family:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>47</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D</string>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>105</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>15</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#A9BA9D</string>
    </property>
    <property name="text">
     <string>ID Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>561</x>
      <y>161</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFamily">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAge">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>280</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditID">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Only letters , numbers and dot</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border : 1px solid silver;
border-radius: 7px;
}</string>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>510</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#QPushButton{
color:silver; 

}
</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
